docs/src/two-syntax-variants-used

```diff
@@ -0,0 +1,12 @@
+This page uses two different syntax variants: the Cython specific ``cdef`` syntax
+and static Cython type declarations in pure Python code,
+following `PEP-484 <https://www.python.org/dev/peps/pep-0484/>`_ type hints
+and `PEP 526 <https://www.python.org/dev/peps/pep-0526/>`_ variable annotations.
+To make good use of the latter, including C data types etc., you need
```

Comment:
```suggestion
To make good use of the latter, including C data types, you need
```

Reply:
Well, firstly, if you really want to, you can also specify C++ and that's it, and secondly, @scoder originally wrote this text, I think this is a question for him.

Reply:
so @scoder say something ;)

Reply:
C, C++ and Python builtin and extension data types. Also, cimports, parallel code, and then some. Thus the etc.